2.Views/WebAppPages/HomePage.cs

```diff
@@ -14,8 +15,35 @@ public class HomePage
     public IClickable LogoutButton =>
         new Button("", new Locator(LocatorType.Id, "ctl00_HeaderTopControl1_LinkButtonLogout"));
 
+    public IClickable ProjectButton =>
+    new Button(
+        "",
+        new Locator(LocatorType.XPath, "//*[@id='MainTable']//tr/td[contains(@class, 'ProjItemContent')][contains(., 'Automation')]")
+    );
+
+    public ITypeable AddToDoInput =>
+        new TextField(
+            "",
+            new Locator(LocatorType.Id, "NewItemContentInput")
+        );
+
     public HomePage()
     {
         //GenericWebDriver.Instance.Navigate().GoToUrl(HostUrl);
     }
+    public void LogoutOffTheApplication()
+    {
+        //GenericWebDriver.Instance.Navigate().GoToUrl(HostUrl);
+    }
+
+    public void SelectProject()
+    {
+        ProjectButton.Click();
+    }
+
+    public void AddItem(string itemName)
+    {
+        AddToDoInput.Type(itemName);
+        AddToDoInput.Type(Keys.Enter);
+    }
```

Comment:
These two methods should be in new classes on the Views/Actions/Items and Views/Actions/Project folders , at least that's what I think we defined for the methods 🧐

Reply:
solved